resources/pgp-keys-map.list

```diff
@@ -212,6 +212,8 @@ doxia:doxia-sink-api:1.0-alpha-4 = noSig
 
 edu.berkeley.cs.jqf             = 0x07665A503E191C320F29C6C8E7C8ECE8FE1536DC
 
+eu.medsea.mimeutil:mime-util:[1.3,2.1.3] = noSig
```

Comment:
I see that all versions don't have signatures https://repo1.maven.org/maven2/eu/medsea/mimeutil/mime-util/
so specific versions range here is not need now

Reply:
My thought was to provide version range, since any future release would likely contain a signature.  Basically, preferring to be as specific as possible to minimize the scope of "noSig".

Reply:
I have rebased and removed the version range.